CMIP6_downscaling_plans.csv

```diff
@@ -515,13 +543,13 @@ MED-12,ENEA,G. Sannino,ENEA-REGb,MPI-ESM1-2-HR,r1i1p1f1,historical,running,2025-
 MED-12,ENEA,G. Sannino,ENEA-REGb,MPI-ESM1-2-HR,r1i1p1f1,ssp370,running,2025-12,
 MED-12,ENEA,G. Sannino,ENEA-REGb,EC-Earth3,r1i1p1f1,historical,running,2025-12,
 MED-12,ENEA,G. Sannino,ENEA-REGb,EC-Earth3,r1i1p1f1,ssp370,running,2025-12,
-MED-12,ICTP-OGS,E. Coppola,RegCM-ES,ERA5,,evaluation,running,2025-03,
```

Comment:
You are dropping OGS here, but @mcreal82 is at the same time trying to register OGS (https://github.com/WCRP-CORDEX/cordex-cmip6-cv/issues/284). Could you clarify the plans with RegCM-ES? (see also #76)